example-app/SantokuApp/src/features/qa-question/components/QuestionList.tsx

```diff
@@ -1,5 +1,20 @@
+import {StyledColumn} from 'bases/ui/common/StyledColumn';
+import {Question} from 'features/backend/apis/model';
 import React from 'react';
 
-export const QuestionList: React.VFC = () => {
-  return <></>;
+import {QuestionListCard} from './QuestionListCard';
+
+export type QuestionListProps = {
+  data?: Question[];
+  navigateToQuestionDetail: (questionId: string) => void;
+};
+
+export const QuestionList: React.VFC<QuestionListProps> = ({data, navigateToQuestionDetail}) => {
+  return (
+    <StyledColumn space="p16">
+      {data?.map(item => (
```

Comment:
Wouldn't a FlatList be nice for an infinite list?

Reply:
There is no infinite scrolling in the basic version, but we changed to FlatList because FlatList has better performance.
e1f487a4577d5a0e0370b18c35c213fe54f00f4d

However, the type resolution of FlatList with restyle applied did not work, so FlatList is used as is.